parsers/dataField/parseMultiESDTNFTTransfer.go

```diff
@@ -0,0 +1,36 @@
+package datafield
+
+import (
+	"github.com/ElrondNetwork/elrond-go-core/core"
+)
+
+func (odp *operationDataFieldParser) parseMultiESDTNFTTransfer(args [][]byte, sender, receiver []byte) *ResponseParseData {
+	responseParse := &ResponseParseData{
+		Operation: core.BuiltInFunctionMultiESDTNFTTransfer,
+	}
+
+	parsedESDTTransfers, err := odp.esdtTransferParser.ParseESDTTransfers(sender, receiver, core.BuiltInFunctionMultiESDTNFTTransfer, args)
+	if err != nil {
```

Comment:
Because we have a call of BuiltInFunctionMultiESDTNFTTransfer 
--> is a valid call:  the name of the function is ok and the arguments are hex-encoded
--> is possible to get an error here because the number of arguments is not correct

Reply:
So we want to signal that a `BuiltInFunctionMultiESDTNFTTransfer` call was made, but it was incorrectly encoded?